### ISSUE TITLE
chinese keyboard does not work 

### ISSUE THREAD

**Issue by WilliamFromTW** [cedrozor/myrtille#293]
glad to see keyboard support multi languages
but i found when i type chinese (one chinese word include two or three key press ),  but it just output every single key not a word,
usually we will press space to tell system "word is ready!" after two or three key press, then system will popup suitable chinese word for us to choose correct one.

![Screen Shot 2021-05-16 at 8 13 05 AM](https://user-images.githubusercontent.com/5213198/118381582-d7291500-b61e-11eb-9582-01b39e241180.png)

apache guacamole works , it seems use native input method not virtual keyboard.

btw , i still can use "text" function to pass chinese , thanks

**Comment by cedrozor** [comment]
Yes, IME composition events are yet to be implemented into Myrtille, as explained into this thread: https://github.com/cedrozor/myrtille/issues/176

Meanwhile, a workaround is to enable IME in the remote session to make multi-bytes per character languages work:

> I finally got it to work with Japanese. Took a lot of trial and error. But with Microsoft IME active on the remote session and Kanji Input is enabled, the phonetic phrases typed from the local keyboard were recognized and MS IME was able to pop-up likely choices for Hiragana characters.
